docs/kyma/03-02-testing.md

```diff
@@ -40,15 +40,19 @@ The following example presents TestDefinition with a container that calls the De
 apiVersion: "testing.kyma-project.io/v1alpha1"
 kind: TestDefinition
 metadata:
-  name: "test-{{ template "fullname" . }}-connection-dex"
+  name: {{ .Chart.Name }}
+  labels:
+    app: {{ .Chart.Name }}-tests
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
 spec:
   template:
     metadata:
       annotations:
         sidecar.istio.io/inject: "false"
     spec:
       containers:
-      - name: "test-{{ template "fullname" . }}-connection-dex"
+      - name: {{ .Chart.Name }}-tests
```

Comment:
I would go with just `test`. Or `main`. It has to be unique at pod scope so let's keep it short and simple.

Reply:
done, used `tests`